reddit_service_websockets/dispatcher.py

```diff
@@ -22,6 +46,14 @@ def __init__(self, metrics):
     def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
+        # Compress the message
+        compressed = make_compressed_frame(message, COMPRESSOR)
```

Comment:
I'm leaning on the side of not doing the "don't bother compressing if it's larger" thing.  If I send the empty payload `{}`, I get a before size of 31 and an after size of 34.  I suspect that by the time we get to the MTU size there's little chance we won't save at least 3 bytes by compressing.